src/helpers/figures/charts/chart_common.ts

```diff
@@ -440,3 +442,13 @@ export function getPieColors(colors: ColorGenerator, dataSetsValues: DatasetValu
 
   return pieColors;
 }
+
+export function truncateLabel(label: string | undefined): string {
```

Comment:
seems to me that it's more 'formatTickValue' that should move to chart_ui_common rather than the other way around as it's clearly 'UI' but that split in the helpers seems already shady atm